### DISCUSSION TITLE
PC, Windows, GNU?

### DISCUSSION THREAD

**Discussion by markszente** [esp-rs/espup#406] [Q&A]
Hello,

Is it possible to use `espup` with GNU as opposed to MSVC on Windows? 

Thanks

**Comment by SergioGasquez** [comment]
Hi! 

Just to make sure, are you referring to the `espup` binary or the Xtensa Rust installed by `espup`? 

If you are referring to the `espup` binary, is not currently available for the `x86_64-pc-windows-gnu` target. It should be very easy to add, still you could build `espup` from source to that target or maybe even try the msvc binary as it might work, but, again, if you require it we can easily add it.

If you are referring to the Xtensa Rust that `espup` installs, it is available for `x86_64-pc-windows-gnu` and you can install it using the `-d, --default-host` arg.
